inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -369,7 +338,7 @@
                         "type": "string"
                     },
                     "cnum": {
-                        "type": "array"
+                        "type": "string"
```

Comment:
@annetteholtkamp this is weird though. Is it due to a bug rather than a feature? How can a document be referring to more than 1 conference? (e.g. http://inspirehep.net/record/1405352 refers to 2 conferences separated by several years!)